integrationTests/testGameHelperFunctions.go

```diff
@@ -26,7 +61,7 @@ func DeployScTx(nodes []*TestProcessorNode, senderIdx int, scCode string) {
 			sndAddr:  nodes[senderIdx].OwnAccount.PkTxSignBytes,
 			data:     scCode + "@" + hex.EncodeToString(factory.IELEVirtualMachine),
 			gasLimit: 100000,
-			gasPrice: MinTxGasPrice,
+			gasPrice: uint64(0),
```

Comment:
why replace everywhere with uint64(0) instead of using a variable MinTxGasPrice?

Reply:
This methods are used for testing game smart contract and before my modifications MinTxGasPrice was 0. After i change the value of MinTxGasPrice variable in tests for game smart contract cannot calculate how many gas  the contract will consume. And after some discussions  tests will use min gas price equals with 0, so in that methods i have to set min gras price with 0.